cmd/orb.go

```diff
@@ -308,13 +308,54 @@ func addOrbElementsToBuffer(buf *bytes.Buffer, name string, namedOrbElements map
 	}
 }
 
+// nolint: unparam
+func addOrbStatisticsToBuffer(buf *bytes.Buffer, name string, stats api.OrbStatistics) {
+	var (
+		err     error
+		encoded []byte
+		data    map[string]int
+	)
+
+	// Roundtrip the stats to JSON so we can iterate a map since we don't care about the fields
+	encoded, err = json.Marshal(stats)
+	if err != nil {
+		panic(err)
+	}
+
+	if err = json.Unmarshal(encoded, &data); err != nil {
+		panic(err)
+	}
+
+	_, err = buf.WriteString(fmt.Sprintf("  %s:\n", name))
+
+	// Sort the keys so we always get the same results even after the round-trip
+	keys := make([]string, 0, len(data))
+	for k := range data {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	for _, key := range keys {
+		value := data[key]
+		_, err = buf.WriteString(fmt.Sprintf("    - %s: %d\n", key, value))
+	}
+
+	// This will never occur. The docs for bytes.Buffer.WriteString says err
+	// will always be nil. The linter still expects this error to be checked.
```

Comment:
Is there a way to disable the linter for a particular case like this, instead of having to write a long comment and effectively 3 lines of dead code?

Reply:
@yundt That's a fair point, we should be able to remove these lines with a single comment `// nolint: errcheck`

Reply:
02b2b93 and 4d209a3 address this.